src/store/mutations.ts

```diff
@@ -0,0 +1,426 @@
+import Vue from 'vue'
+import { StateIF } from '@/interfaces/state-interface'
+import {
+  AnalysisJSONI,
+  ApplicantI,
+  ConversionTypesI,
+  LocationT,
+  NameRequestI,
+  SelectOptionsI,
+  StatsI,
+  SubmissionTypeT, WaitingAddressSearchI
+} from '@/interfaces/models'
+
+export const clearErrors = (state: StateIF) => {
+  state.stateModel.newRequestModel.errors = []
+}
+
+export const mutateActingOnOwnBehalf = (state: StateIF, actingOnOwnBehalf: boolean) => {
+  state.stateModel.newRequestModel.actingOnOwnBehalf = actingOnOwnBehalf
+}
+
+export const mutateAddressSuggestions = (state: StateIF, addressSuggestions: object | null) => {
+  if (!addressSuggestions) {
+    state.stateModel.newRequestModel.actingOnOwnBehalf = null
+    return
+  }
+  state.stateModel.newRequestModel.addressSuggestions = Object.assign([], addressSuggestions)
+}
+
+export const mutateAnalysisJSON = (state: StateIF, analysisJSON: AnalysisJSONI) => {
+  state.stateModel.newRequestModel.analysisJSON = analysisJSON
+}
+
+export const mutateApplicant = (state: StateIF, appKV: any) => {
+  if (Array.isArray(appKV)) {
+    for (let address of appKV) {
+      state.stateModel.newRequestModel.applicant[address.name] = address.value
```

Comment:
This is one of those properties that is sometimes undefined in Prod.

This is probably also why it's so slow to populate the address fields.

I'm not sure that I want to refactor this atm, but let's keep it in mind.

Reply:
Absolutely! 
Leave all these comments open so we can look back :)